ncm-mysql/src/main/pan/components/mysql/schema.pan

```diff
@@ -35,8 +35,8 @@ function component_mysql_check_db_script = {
 };
 
 type component_mysql_db_user = {
-  'password'  : string
-  'rights'    : string[]
+  'password'  : string with match(SELF, '^[^\\]+$')
+  'rights'    : string[] = list('ALL PRIVILEGES')
```

Comment:
is `ALL PRIVILIGES` really the default in mysql? it seems very very wrong (and maybe we should settle for something lower; like readonly?)